docs/monitoring/flow.rst

```diff
@@ -0,0 +1,68 @@
+
+North Service Data Flow
+-----------------------
+
+The north services also collect statistics, in this case we can get the total number of readings sent to all north systems or the number sent to each north system. Unlike the south services both of these are always collected and so they can be utilised in all cases.
+
+Statistics Mechanism
+--------------------
+
+Statistics are gathered as pure counters that increment as data traversing the pipeline or whatever the statistic is monitoring, this is not particularly useful when trying to monitor rates. This is however another options, the statistics collector is a task that runs at periodic intervals and takes the raw statistics to create a statistics history table. The statistics collector is run by default every 15 seconds, although this interval can be altered via the configuration. The statistics table will therefore record the increment in each statistic every 15 seconds. This allows us to easily obtain a rate of change for any statistic gathered by the system. It is these statistics history entries that are shown by the Flog dashboard, thus it shows the number of readings ingested every 15 seconds by default.
+
+Both the raw statistics and the statistics history can be used as data feeds into the internal Fledge notification service. This means that we can use the Fledge mechanisms for capturing events and sending notifications to many destinations to monitor the flow rates of our south and north services.
+
+Monitor Low North Flow Rate
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As an example we will setup a notification on the flow rate north dropping below a low water mark. For this we will use the threshold notification rule and use the statistics history as the source of data. Let's assume we want to be notified when we send less than 400 readings north in a any given minute. Given our default 15 second statistics collection schedule we can either change that to be once a minute or we can set our threshold lower to 100 readings in every 15 seconds. For this example we will do the later.
+
++--------------------+
+| |MonitorNorthRate| |
++--------------------+
+
+When can then choose one of Fledge's many supported delivery plugins to have this notification sent to the destination that we require. For the purpose of this example we will use the ZenDesk plugin to raise a problem ticket to have the cause of the issue investigated.
+
++------------------+
+| |MonitorZendesk| |
++------------------+
+
+We can then decide how we want this to trigger, it is probably most appropriate to have the alert trigger first when we drop below that threshold and then be cleared when we rise above that threshold. We also do not wish to have it re-trigger again too soon, so we will set the re-trigger interval to 3600 seconds, or every hour. This way we do not raise excessive number of tickets.
```

Comment:
- Start new sentence at "It is probably most appropriate..."
- "This way we do not raise excessive number of tickets." -> "This way we do not raise excessive numbers of tickets."